### PR TITLE
Add Spoonacular API

### PR DESCRIPTION
**Pull request by RahulBansal123**
Adding more APIs to help developers from all over the world.

Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [ ] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [ ] Your additions are ordered alphabetically
- [ ] Your submission has a useful description
- [ ] The description does not end with punctuation
- [ ] Each table column should be padded with one space on either side
- [ ] You have searched the repository for any relevant issues or pull requests
- [ ] Any category you are creating has the minimum requirement of 3 items
- [ ] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by SitramSoft** [comment]
Shouldn't the commits should be squashed together into one with a more meaningful description?

**Comment by saeedkkssaa** [comment]
F

File: README.md [hunk 1]
```diff
@@ -537,6 +538,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Deepcode](https://www.deepcode.ai/docs/Overview%252FOverview) | AI for code review | No | Yes | Unknown |
 | [Dialogflow](https://dialogflow.com) | Natural Language Processing | `apiKey` | Yes | Unknown |
 | [Keen IO](https://keen.io/) | Data Analytics | `apiKey` | Yes | Unknown |
+| [Microsoft](https://azure.microsoft.com/en-in/services/cognitive-services/face/#features) | AI for face detection | No | Yes | Unknown |
```
Comment (PawelBorkar) [inline README.md:541]: I think Azure is having OAuth

**Comment by matheusfelipeog** [comment]
Hi @RahulBansal123 😄

Thank you for your contribution. ✋

I noticed that there is a conflict in your pull request and you deleted the fork from this repository, so it will not be possible to resolve it. Consider creating a new pull request if you still want to add this APIs to be reviewed and added to our listing.

**Pull request closed without merging**